src/mixins/filing-template-mixin.ts

```diff
@@ -134,8 +135,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
       // make a copy so we don't change original array
       let parties = cloneDeep(this.getPeopleAndRoles)
 
-      // add completing party
-      parties = this.addCompletingParty(parties)
+      // add completing party (client error correction only)
+      if (this.isClientErrorCorrection) {
+        parties = this.addCompletingParty(parties)
```

Comment:
As per discussion in 13194, only include Completing Party if this is a client-error correction filing.